libraries/rush-lib/src/cli/parsing/SelectionParameterSet.ts

```diff
@@ -183,6 +186,13 @@ export class SelectionParameterSet {
         ' belonging to VERSION_POLICY_NAME.' +
         ' For details, refer to the website article "Selecting subsets of projects".'
     });
+
+    this._subspaceParameter = action.defineStringParameter({
+      parameterLongName: '--subspace',
+      argumentName: 'SUBSPACE_NAME',
+      description:
+        '(EXPERIMENTAL) Specifies a Rush subspace to be installed. Requires the feature to be enabled in subspaces.json.'
```

Comment:
Mention what the property name in the config file needs to be set.

Reply:
Added